Resources/Prototypes/Entities/Objects/Shields/shields.yml

```diff
@@ -441,7 +444,7 @@
               acts: [ "Destruction" ]
```

Comment:
Also unreachable as the other threshold is 160 and breaks it, unless you instantly take the extra 20 and get unlucky with RNGesus (since these are likely not executed in a deterministic order).